### ISSUE TITLE
[FEATURE REQUEST] Расписание учителя

### ISSUE THREAD

**Issue by ValeryVerkhoturov** [Ninja-Official/rtu-mirea-mobile#69]
**Проблема**
В [README](https://github.com/Ninja-Official/rtu-mirea-mobile/blob/master/README.md) указано, что приложение создано для студентов и преподавателей. Инструментов для преподавателей нет. Предлагаю возможность просмотра расписания преподавателя. Зачем?
- инструмент для преподавателя (первый и самый необходимый, ни один преподаватель не будет искать свою фамилию в расписаниях огромного кол-ва групп)
- поиск преподавателя для студентов (в начале и конце семестра встретил много должников, ищущих преподавателей)

**Решение**
Возможность в поле ввода группы ввести фамилию преподавателя и увидеть его расписание

**Comment by 0niel** [comment]
Да, спасибо, обязательно сделаем. Только есть несколько существенных проблем:
1. В расписании не всегда ставят имя преподавателя.
2. В расписании иногда ставят вместо имени преподавателя какой-то бред.
3. Есть преподаватели с одинаковой фамилией и инициалами. А в расписании пишут имя в формате: "Фамилия И. О."
